gnucash/gnome/dialog-price-edit-db.c

```diff
@@ -782,6 +792,8 @@ gnc_prices_dialog_create (GtkWidget * parent, PricesDialog *pdb_dialog)
         gtk_widget_grab_default (button);
         gtk_widget_grab_focus (button);
 
+        button = GTK_WIDGET(gtk_builder_get_object (builder, "help_button"));
```

Comment:
My feeling says, it should be earlier in the list.
@Bob-IT Is there a recommend order?

Reply:
This line is not required.